pkg/controller/windowsmachine/windows/windows.go

```diff
@@ -216,6 +222,25 @@ func (vm *windows) ConfigureHybridOverlay(nodeName string) error {
 	return nil
 }
 
+func (vm *windows) ConfigureWindowsExporter() error {
+	// configures service to collect and expose metrics at endpoint with default port :9182 and default URL path /metrics
+	windowsExporterServiceArgs := "--collectors.enabled " +
+		"\"cpu,cs,logical_disk,net,os,service,system,textfile,container,memory\""
+
+	windowsExporterService, err := newService(WindowsExporterPath, windowsExporterServiceName, windowsExporterServiceArgs)
+	if err != nil {
+		return errors.Wrap(err, "error creating service object")
+	}
+	if err := vm.createService(windowsExporterService); err != nil {
```

Comment:
this can now follow the pattern for other services like hybrid-overlay and kube-proxy, where we now first check if the service exists, and only create if  it does not.